.github/workflows/labels.yml

```diff
@@ -0,0 +1,44 @@
+name: Add Label
+
+on:
+  pull_request:
+    types: [ edited, labeled, opened, reopened ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  no_ticket:
+    name: "Label whether ticket linked from title"
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: kaisugi/action-regex-match@v1.0.1
+        id: regex-match
+        name: "Check title"
+        with:
+          text: ${{ github.event.pull_request.title }}
+          regex: '#[0-9]+ '
+          flags: gm
+
+      - run: gh pr edit "$NUMBER" --add-label "$LABELS"
```

Comment:
Would it make sense to re use actions to add a remove labels? For example
https://github.com/actions-ecosystem/action-add-labels
https://github.com/actions-ecosystem/action-remove-labels

Reply:
Ah I originally tried this and couldn't get it working but I can try again and report back

Reply:
It works, I think I had a permission issue before 👍